src/data/nat/choose/sum.lean

```diff
@@ -103,6 +103,17 @@ begin
   simpa [sum_range_choose_halfway n] using t
 end
 
+lemma four_pow_le_two_mul_add_one_mul_central_binom (n : ℕ) :
+  4 ^ n ≤ (2 * n + 1) * choose (2 * n) n :=
+begin
+  calc 4 ^ n = (1 + 1) ^ (2 * n) : by norm_num [pow_mul]
+  ...        = ∑ m in range (2 * n + 1), choose (2 * n) m : by simp [add_pow 1 1 (2 * n)]
```

Comment:
Not a necessary change, but just as an fyi: `simp` is able to infer arguments implicitly, so you aren't required to pass them.
```suggestion
  ...        = ∑ m in range (2 * n + 1), choose (2 * n) m : by simp [add_pow]
```